staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -257,6 +257,7 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		(func() ([]byte, []byte))(s.proxyCurrentCertKeyContent),
 		s.serviceResolver,
 		c.GenericConfig.EgressSelector,
+		c.GenericConfig.LifecycleSignals.HasBeenReady.Signaled(),
```

Comment:
for this we can add a method and avoid changing all references to LifecycleSignals. wdyt?

Reply:
sure, np